camayoc/tests/qcs/test_credentials.py

```diff
@@ -132,12 +134,12 @@ def test_negative_update_to_invalid(shared_client, cleanup):
     :expectedresults: Error codes are returned and the host credentials are
         not updated.
     """
-    ssh_keyfile = Path(uuid4())
-    ssh_keyfile.touch()
+    sshkeyfile = Path(os.path.join(isolated_filesystem, uuid4()))
+    sshkeyfile.touch()
```

Comment:
tmp files were being created in the directory I was executing the tests so I did this

Reply:
If I put a `pdb` break in the `isolated_filesystem` utils function, in the fixutre, and in one of the tests, and query `os.environ['PWD']` in all contexts it shows me that I am still in the directory where I executed the test.  So I guess this si the same context that the Path object is probably getting its PWD.

But the tmp directory IS generated by `isolated_filesystem` and when I join the path this way, the sshkeyfile gets created in that directory, and after the test finishes, the tmp directory is destroyed. 

That is all know at this point.

Reply:
The `Path` object is supposed to be relative to the CWD as returned by `os.getcwd`. I did this on an interactive session:

```python
In [1]: import pathlib

In [2]: import os

In [3]: os.getcwd
Out[3]: <function posix.getcwd>

In [4]: os.getcwd()
Out[4]: '/home/elyezer/code/camayoc'

In [5]: os.chdir('/tmp')

In [7]: path = pathlib.Path('.')

In [8]: path.resolve()
Out[8]: PosixPath('/tmp')

In [10]: test = pathlib.Path('test')

In [11]: test.touch()

In [12]: !ls /tmp/test
/tmp/test

In [13]: os.getcwd()
Out[13]: '/tmp'
```

So if the files are being created on where you are running py.test then there is an issue that we need to figure out. The isolated_filesystem is changing the directory to the new created temp directory.

Can you please investigate more?

Reply:
Wait, I have found the root cause, there are tests creating files that are no changing directory. I am going to send a PR to fix that.

Reply:
Sent https://github.com/quipucords/camayoc/pull/80 to fix it

Reply:
I reviewed and approved it. Thanks, I rebase once merged.